### PR TITLE
AB#8514 PLAY! Media Native - Flat List performance optimization

### PR DESCRIPTION
**Pull request by chpa7**

### CODE REVIEW

**Comment by sc-nelikostadinova** [comment]
When I open the app initially events load correctly but the fist time I go to the athletes or media tab, nothing loads until I interact with the screen.

**Comment by chpa7** [comment]
> When I open the app initially events load correctly but the fist time I go to the athletes or media tab, nothing loads until I interact with the screen.

You mean you don't see anything on Athlete/ Media listing screens initially? Probably something iOS-related, I'm not getting this behavior on Android, everything loads correctly.

File: play-media-native/components/Listing/Listing.tsx [hunk 1]
```diff
@@ -115,6 +115,9 @@ export const Listing = ({
           />
         }
         onScroll={onScroll}
+        removeClippedSubviews
```
Comment (sc-nelikostadinova) [inline play-media-native/components/Listing/Listing.tsx:118]: I read a bit and it seems removeClippedSubviews has an issue when used with tabs on iOS. Please add this prop only on android devices. I don't know if it's going to lower performance, but I don't think there's a fix for it.

**Review by sc-nelikostadinova — CHANGES_REQUESTED** [review]

File: play-media-native/components/Listing/Listing.tsx (hunk 1 above)
Comment (chpa7) [reply play-media-native/components/Listing/Listing.tsx]: Removing it altogether as it does not offer any huge performance improvement

**Pull request merged**